modules/ROOT/pages/dialog-configuration.adoc

```diff
@@ -62,3 +67,46 @@ For more information on the `+dialogApi+` object that is passed to some of the c
 Each of the event callback functions - `+onAction+`, `+onSubmit+`, `+onCancel+`, `+onChange+`, `+onClose+`, and `+onTabChange+` - are shared between all dialog components that may trigger them. For example, *Custom* type footer buttons and dialog panel buttons all trigger `+onAction+`. Therefore, callback functions that may be triggered by multiple components are passed an object (called `+details+` above) that contains the `+name+` of the component that triggered the event.
 
 Any callback function that is not passed a `+details+` object assumes that the dialog will only contain one component which can trigger it or that it does not matter if the function is triggered by multiple components. For example, `+onSubmit+` is only triggered when a user clicks on a *Submit* type footer button, and {productname} assumes that a dialog will only have one *Submit* type button. In comparison, `+onCancel+` and `+onClose+` are both triggered by clicking the `+X+` button in the top right of a dialog or by clicking a *Cancel* type footer button. These two buttons have the same functionality, and therefore {productname} does not differentiate between them.
+
+== Dialog position
+
+By default a dialog is shown as a modal in the center of the editor viewport.
+
+This presented position can be changed by providing a second argument, with an appropriate value, to `editor.windowManager.open()`.
+
+[[inline]]
+=== `+inline+`
+
+In a dialog configuration, the `+inline+` option sets the dialog’s presented position to one of two possiblities.
```

Comment:
The configuration for inline dialog is passed as a second argument to the `windowManager.open`, it's not part of the `dialog` configuration, so should be in the second argument. Maybe:

```suggestion
The `+inline+` option passed to the `editor.windowManager.open` sets the dialog’s presented position to one of two possiblities.
```